lib/src/lifecycle_module.dart

```diff
@@ -67,9 +67,9 @@ abstract class LifecycleModule {
   /// Calls the onLoad() method, which can be implemented on a Module.
   /// Executes the willLoad and didLoad event streams.
   Future load() async {
-    _willLoadController.add(this);
+    await _willLoadController.add(this);
     await onLoad();
-    _didLoadController.add(this);
+    await _didLoadController.add(this);
```

Comment:
don't know why it should work this way, but it definitely does...an alternative that also works is to replace the async / await syntax with completers:
https://github.com/Workiva/w_module/pull/41
any votes on which approach to use?